run-jetmon.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+mkdir -p logs
```

Comment:
I'd consider adding the logs and stats related directories and files on build instead :) 
I'm still not sure these should be part of the image build though or it's better to create them on our local envs.
Same with certs.

Reply:
I fixed that up in my second commit where I just ensured that both directories are part of the repo. At this point, the script just acts as a sanity check to ensure that the directories are present. I think it works better set up this way (as part of the repo by default) as opposed to some `Dockerfile` build step as the build process could result in directories that exist in the docker filesystem but not the local filesystem, which would make testing and log inspection more difficult.

We don't have to stick with the logic I added in the `run-jetmon.sh` script (or keep the script at all). I mainly wanted to throw it out there to show that we can launch using a script that contains whatever logic we want rather than being stuck with just running the `npm` command to launch the container.